docs/reference/configuration.md

```diff
@@ -722,6 +723,40 @@ If `True`, don't fail if the version string to be replaced is not found in the f
 
 if `True`, don't fail if the configured file is missing.
 
+### valid_bumps
+
+::: field-list
+
+    required
+    : No
+    
+    default
+    : all version parts
+    
+    type
+    : list of strings
+
+The `valid_bumps` file configuration allows you to control when this file is changed. When a `bump <version component>` command is issued, this file is changed only if the version component is in this list and not in [`invalid_bumps`](#invalid_bumps). The [parse](#parse) configuration defines version components.
+
+The default value, or an empty list, includes all version components.
+
+### invalid_bumps
+
+::: field-list
+
+     required
+     : No
+     
+     default
+     : `[]`
+     
+     type
+     : list of strings
+
+The `invalid_bumps` file configuration allows you to control when this file is changed by exclusion. When a `bump <version component>` command is issued, this file is only changed if the version component is *not in this list.* The [parse](#parse) configuration defines version components.
```

Comment:
They are always combined because the included bumps are always defined. Declaring both of them doesn't make much sense, but it works. I'll try and adjust the docs accordingly.